projects/demo/src/app/highlighter.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import { codeToHtml } from 'shiki';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class HighlighterService {
+  async highlight(code: string, lang: string): Promise<string> {
+    return codeToHtml(code.toString().trim(), {
```

Comment:
Why the toString()?

Reply:
It was in the prism service, I pasted without thinking 🤷👍